src/utils/styles.js

```diff
@@ -7,6 +7,10 @@ const reSkew = /((^|\s)skew(x|y)?\()\s*(\-?[\d]+)(deg|rad|grad)(,\s*(\-?[\d]+)(d
 
 export default {
 
+  merge() {
+    return ImmutabilityHelper.merge.apply(this, arguments);
```

Comment:
it's not necessary, We should remove mixins in the future anyway, so I wouldn't bother :D